src/odemis/acq/stream/_static.py

```diff
@@ -390,7 +393,7 @@ def _setBackground(self, data):
 
         # check it's compatible with the data
         data = img.ensure2DImage(data)
-        arpole = data.metadata[model.MD_AR_POLE] # we expect the data has AR_POLE
+        arpole = data.metadata[model.MD_AR_POLE]  # we expect the data has AR_POLE
```

Comment:
maybe you can also import MD_AR_POLE in the beginning? Otherwise, you sometimes have to use the model.MD...
prefix, and sometimes not, which might be confusing